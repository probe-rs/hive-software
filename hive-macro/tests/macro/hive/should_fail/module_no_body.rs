#[hive_macro::hive]
pub mod tests;

fn main() {}
