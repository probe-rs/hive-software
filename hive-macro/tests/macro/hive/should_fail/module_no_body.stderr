error[E0658]: non-inline modules in proc macro input are unstable
 --> tests/macro/hive/should_fail/module_no_body.rs:2:1
  |
2 | pub mod tests;
  | ^^^^^^^^^^^^^^
  |
  = note: see issue #54727 <https://github.com/rust-lang/rust/issues/54727> for more information
  = help: add `#![feature(proc_macro_hygiene)]` to the crate attributes to enable

error: Top level module for Hive must have a body
 --> tests/macro/hive/should_fail/module_no_body.rs:2:1
  |
2 | pub mod tests;
  | ^^^^^^^^^^^^^^
  |
  = help: Add a module body
  = note: example:

          #[hive]
          pub mod tests {
          // Testfunctions etc...
          }
